index.html

```diff
@@ -3,10 +3,166 @@
 
 <head>
     <title>CV</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed&display=swap" rel="stylesheet">
 </head>
 
 <body>
-    <div id="root"></div>
+    <div id="root" class="container">
+        <header>
+            <div class="content">
+                <div class="photo-container">
```

Comment:
Don't forget about BEM. `photo-container` is block and `photo` inside is an element